src/usng.ts

```diff
@@ -28,1352 +28,1380 @@
 
 /*global define,Math,parseInt,parseFloat*/
 
+type LL = {
+  lat: number,
+  lon: number
+};
+
+type BBox = {
+  north: number,
+  south: number,
+  east: number,
+  west: number
+};
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atanh
+const atanh = Math["atanh"] || function(x) {
+  return Math.log((1+x)/(1-x)) / 2;
+};
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON
+const epsilon = typeof Number["EPSILON"] === "undefined" ? Math.pow(2, -52) : Number["EPSILON"];
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot
+const hypot = Math["hypot"] || function (x, y) {
+  // https://bugzilla.mozilla.org/show_bug.cgi?id=896264#c28
+  var max = 0;
+  var s = 0;
+  for (var i = 0; i < arguments.length; i += 1) {
+    var arg = Math.abs(Number(arguments[i]));
+    if (arg > max) {
+      s *= (max / arg) * (max / arg);
+      max = arg;
+    }
+    s += arg === 0 && max === 0 ? 0 : (arg / max) * (arg / max);
+  }
+  return max === 1 / 0 ? 1 / 0 : max * Math.sqrt(s);
```

Comment:
❓ Is this the same as
`return max === NaN ? NaN : max * Math.sqrt(s);`
If so, wouldn't that be clearer?

Reply:
I was very curious about this line as well, but I am hesitant to make any changes to it because this code is ES5 polyfill from the link provided atop the function.